blaze-core/src/test/scala/org/http4s/blazecore/util/CatsEffect.scala

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2013-2020 http4s.org
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.http4s.blazecore.util
+
+import cats.effect.std.Dispatcher
+import cats.effect.{Async, Resource, Sync}
+import org.specs2.execute.{AsResult, Result}
+
+import scala.concurrent.duration.{Duration, _}
+
+/** copy of [[cats.effect.testing.specs2.CatsEffect]] adapted to cats-effect 3
```

Comment:
This would be a good issue on the cats-effect-testing repo.

Until then, I think we could put it in our testing package and maybe use it in multiple modules.

Reply:
https://github.com/http4s/http4s/pull/3894/commits/52caa170f4c4859050f7876492a112c6f40c2856